assignment.js

```diff
@@ -65,6 +69,13 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+let numbers = [3, 48, 29, 12, 18]
+
+let currentLargestNumber = numbers.reduce(function (a, b) {
+    return Math.max(a, b)
+});
+
+console.log(currentLargestNumber)
 
```

Comment:
@OcielC Nice work here using the reduce function in combination with the Math.max method to return the largest value. Here is another approach to the problem [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L73)